src/Config/Parser/MetadataParser/MetadataParser.php

```diff
@@ -134,6 +134,8 @@ private static function processFile(SplFileInfo $file, ContainerBuilder $contain
             }
 
             return $preProcess ? self::$map->toArray() : $gqlTypes;
+        } catch (ReflectionException $e) {
```

Comment:
I'm not sure about this. What if there is other kinds of `ReflectionException` ? They would get silenced, wont they?
Are we 100% sure that this `catch` can only be trigger by invalid files?

Reply:
reflection exception will happen where it fails to reflect using ReflectionClass. That will usually mean that the file is invalid or contains no class. 

So we have 2 situations
- when the file contains no class - this is the behaviour w want.
- if the file is invalid, the php processor will be responsible of throwing the exceptions. 

There could be a bigger feature - if we would want to validate the annotations / attributes at preParse stage. 

But i don't believe it's in the scope of the current PR

I tried an alternative with class_exists, but it will cause regression on other scenarios in the unit tests (i didn't take notes of the regressions)